std/internal/cstring.d

```diff
@@ -200,7 +201,7 @@ nothrow @nogc @system unittest
     assert(tempCString(abc[].byChar)[] == abc);
 }
 
-// Bugzilla 14980
+// https://issues.dlang.org/show_bug.cgi?id=14980
```

Comment:
Note check this and the other change against git blame. 1498 and 14980 are a suspicious coincidence.

Reply:
https://issues.dlang.org/show_bug.cgi?id=14980
```
commit 156bf68ef7008343755ab450facdd0e5604f6fac
Author: k-hara <k.hara.pg@gmail.com>
Date:   Mon Aug 31 21:57:01 2015 +0900

    fix Issue 14980 - getAddressInfo(null) broken
```
156bf68ef7008343755ab450facdd0e5604f6fac

Reply:
This is what I meant: https://github.com/dlang/phobos/pull/7079/files

The prior change was accidentally changed to 1498 instead of 14980 when it was moved. Please fix in the new PR.